packages/controllers/src/snaps/SnapController.ts

```diff
@@ -45,6 +45,16 @@ export const controllerName = 'SnapController';
 export const SNAP_PREFIX = 'wallet_snap_';
 export const SNAP_PREFIX_REGEX = new RegExp(`^${SNAP_PREFIX}`, 'u');
 
+// APIs exposed by default to the Snap without needing permissions
+export const DEFAULT_EXPOSED_APIS = [
+  'atob',
+  'btoa',
+  'TextEncoder',
+  'TextEncoderStream',
+  'TextDecoder',
+  'TextDecoderStream',
```

Comment:
That's a good time to move default apis out of iframe execution package here

Reply:
Also, add WebAssembly here, it's secure and shouldn't require additional endowments. (CC @rekmarks)

Reply:
Waiting to hear back from @rekmarks on his opinion on having WebAssembly exposed by default. What do you mean with moving APIs out of the iframe execution package? I couldn't find any references to that 🤔

Reply:
Let's add `WebAssembly` to the list of defaults, however it should not include the `initializeStreaming` or `compileStreaming` properties since they are not available in Node.

To implement this, we should add the string "WebAssembly" to this list, but we must construct a `WebAssembly` object in the execution environment that only includes our desired properties, in a similar fashion to what we do with `setTimeout`.

Reply:
Let's add `WebAssembly` to the list of defaults, however it should not include the `initializeStreaming` or `compileStreaming` properties since they are not available in Node.

To implement this, we should add the string "WebAssembly" to this list, but we must construct a `WebAssembly` object in the execution environment that only includes our desired properties, in a similar fashion to what we do with `setTimeout`.

Reply:
@rekmarks I don't think it makes sense to add it to the list and also do a custom implementation. Because default APIs are added as is currently.

Reply:
Added to `BaseSnapExecutor` though!